webapp/components/Map.tsx

```
@@ -1,27 +1,422 @@
 "use client";
 
-import { useEffect, useRef } from "react";
-import maplibregl from "maplibre-gl";
+import { useEffect, useMemo, useRef, useState } from "react";
+import maplibregl, {
+  Feature,
+  GeoJSONSourceSpecification,
+  LngLat,
+  Map,
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)